src/PKSim.Infrastructure/Reporting/Summary/UsedBuildingBlocksReportBuilder.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Collections.Generic;
 using PKSim.Core.Model;
-using PKSim.Core.Reporting;
+using OSPSuite.Core.Reporting;
```

Comment:
@rwmcintosh This change is a bit of noise maker. Why did we move this to core? This is really PKSim specific stuff I think

Reply:
yep. Let's move it back and move it if we need it (neveR)